README.md

```diff
@@ -175,6 +173,7 @@ Have fun creating and using this plugin and the Nexus platform, we are glad to h
 
 Looking to contribute to our code but need some help? There's a few ways to get information:
 
+* P2 repositories Sonatype help documentation [Sonatype Help](https://help.sonatype.com/repomanager2/p2-repositories-equinox-p2)
```

Comment:
Should this be "repomanager3"?

Reply:
Done e402d7282c7d5e04f122b32b85ea8422c3cd9fd8